open-source-policy/practice.md

```diff
@@ -1,33 +1,33 @@
 # Practicing our open source policy #
 
-This document is meant to give specific team guidance on putting our [open
-source policy](policy.md) into practice.
+We have an "open-by-default" development policy. This document is meant to
+give specific team guidance on putting our [open source policy](policy.md)
+into practice.
 
-* CISA releases software into the [international public domain](#public-domain).
-* All team members should feel empowered to contribute back to outside open
-  source projects.
+* CISA releases software into the [international public domain](#public-domain)
+* Contribute back to outside open source projects whenever possible.
 * We [develop our software in the open](#working-in-public), while also
-  [protecting sensitive information](#protecting-sensitive-information).
+  [protecting sensitive information](#protecting-sensitive-information)
 * There are [narrow, documented exceptions](#exceptions) where we may delay or
-  withhold source code.
+  withhold source code
 
-CISA team members should work with the strong presumption that all of their code
-will be public, throughout and after development.
+CISA team members should work with the strong presumption that all of their
+code will be public, both during and after development.
 
 Before deciding to delay or withhold the release of source code, consult with
 the team and be prepared to publicly document this exception.
 
 ## Public domain ##
 
 [By law](http://www.law.cornell.edu/uscode/text/17/105), works of the United
-States government are not copyrightable in the US, and so are public domain. But
-by default, US government works **are** copyrightable internationally, and so
-CISA intentionally waives this copyright abroad using [Creative Commons Zero
+States government are not copyrightable in the US, and so are public domain.
+But by default, US government works **are** copyrightable internationally, and
+so CISA intentionally waives this copyright abroad using [Creative Commons Zero
 (CC0) 1.0](https://creativecommons.org/publicdomain/zero/1.0/).
 
-There are potentially other cases where copyright is involved: where contractors
-produce the work, or where work was otherwise originally performed not in the
-capacity of a US government employee.
+There are potentially other cases where copyright is involved: where
+contractors produce the work, or where work was otherwise originally performed
+not in the capacity of a US government employee.
```

Comment:
Is "not in the" :arrow_right: "outside the" less awkward here?

Reply:
Good change - updated in in 267f33866baf7df7d71137c503c07968d32d75e2